engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetViewportToSizeOf.java

```diff
@@ -95,7 +81,6 @@ public String toString() { // TODO: used for logging purposes at the moment, inv
     }
 
     private FBO getFbo() {
-        return defaultDynamicFBO != null ? defaultDynamicFBO.getFbo() : frameBuffersManager.get(fboName);
-
+        return frameBuffersManager.get(fboName);
```

Comment:
No, it's used in 2 functions, across a total of 6 different calls (can be reduced to 4, but no less). In fact, if anything, I recommend we simply store the fbo instead of retrieving it every time (although that will force us to subscribe to update), although I'm not sure how lean this process has to be.